generator/operation.go

```diff
@@ -950,25 +977,37 @@ func (b *codeGenOpBuilder) MakeParameter(receiver string, resolver *typeResolver
 			Enum:             param.Enum,
 		}
 
+		res.ZeroValue = res.resolvedType.Zero()
+
 		if param.Items != nil {
+			if res.CollectionFormat == "" {
+				// When not defined,CollectionFormat defaults to csv
+				res.CollectionFormat = "csv"
```

Comment:
I'm not a fan of making it explicit like this because it risks the chance of breaking a default and sprinkles this handling all over the code. Whereas handling an empty value as default in the things that deal with strfmt seems wiser

Reply:
I understand, this means handling the default in templates, everywhere we use Split/JoinByFormat
Is it what you want? Even more logics in templates?

Reply:
no Splt and JoinByFormat already deal with the empty string case as csv

Reply:
OK. Will test this, though.